Warn users about file carving size limits (#47376)
fleetdm/fleet: docs/REST API/rest-api.md

- [Get carve](#get-carve)
- [Get carve block](#get-carve-block)

File carving allows you to request files (and sets of files) and their full contents from hosts.

To initiate a file carve using the Fleet API, use the [live report](#run-live-report) endpoint to run a query against the `carves` table.

> Any failure when uploading a file block (like a network error) will result on a failed carved file. Block uploads are retried up to three times before failing.

Reviewer: ```suggestion
Any failure when uploading a file block (like a network error) will result in a failed carved file. Block uploads are retried up to three times before failing.
```

- [Get carve](#get-carve)
- [Get carve block](#get-carve-block)

> > TODO: Warning on size of carve

Author: ```suggestion
```

> File carving allows you to request files (and sets of files) and their full contents.

Author: ```suggestion
File carving allows you to request files (and sets of files) and their full contents from hosts.
```

> To initiate a file carve using the Fleet API, use the [live query](#run-live-query) endpoint to run a query against the `carves` table.

Author: ```suggestion
To initiate a file carve using the Fleet API, use the [live report](#run-live-report) endpoint to run a query against the `carves` table.
```

Any failure when uploading a file block (like a network error) will result on a failed carved file. Block uploads are retried up to three times before failing.

> To learn more about executing a file carve in Fleet, head to the [file carving guide](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/product-groups/orchestration/file-carving.md).

Author: ```suggestion
To learn more about executing a file carve in Fleet, head to the [file carving guide](https://fleetdm.com/guides/file-carving).
```
